src/Microsoft.IIS.WebManager/app/common/dynamic.component.ts

```diff
@@ -89,7 +83,7 @@ export class DynamicComponent implements OnInit {
         }
 
         let vRef = this.vcRef;
-        return DynamicComponent.CreateModuleWithComponentFactories(this.compiler, this._moduleName, this._modulePath)
+        return ComponentLoader.LoadAsync(this.compiler, this.module)
```

Comment:
Angular CLI statically analyze which code do bring in during compile time. I cannot pass a expression to System.Import function anymore. The parameter needs to be a string literal for the code to work